src/llvm/ext/find-llvm-config.ps1

```diff
@@ -0,0 +1,24 @@
+$LLVM_CONFIG = $env:LLVM_CONFIG -or (Get-Command "llvm-config" -ErrorAction SilentlyContinue)
```

Comment:
`-or` always returns a boolean, so if `%LLVM_CONFIG%` was already set, this would become `True`.